content/docs/rfcs/61/README.md

```diff
@@ -0,0 +1,365 @@
+4. Member node receives waku message ([14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/)) that contains the metadata magnet link from the special channel
+5. Member node extracts the magnet link from the waku message and passes it to torrent client
+6. Member node downloads [message archive index](#message-history-archive-index) file and determines which message archives are not downloaded yet (all or some)
+7. Member node fetches missing message archive data via torrent
+8. Member node unpacks and decompresses message archive data to then hydrate its local database, deleting any messages for that community that the database previously stored in the same time range as covered by the message history archive
+
+# Storing live messages
+
+For archival data serving, the control node MUST store live messages as [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/). This is in addition to their database of application messages. This is required to provide confidentiality, authenticity, and integrity of message data distributed via the BitTorrent layer, and later validated by community members when they unpack message history archives.
+
+Control nodes SHOULD remove those messages from their local databases once they are older than 30 days and after they have been turned into message archives and distributed to the BitTorrent network.
+
+## Exporting messages for bundling
+
+Control nodes export Waku messages from their local database for creating and bundling history archives using the following criteria:
+
+- Waku messages to be exported MUST have a `contentTopic` that match any of the topics of the community channels
+- Waku messages to be exported MUST have a `timestamp` that lies within a time range of 7 days
+
+The `timestamp` is determined by the context in which the control node attempts to create a message history archives as described below:
+
+1. The control node attempts to create an archive periodically for the past seven days (including the current day). In this case, the `timestamp` has to lie within those 7 days.
+2. The control node has been offline (control node's main process has stopped and needs restart) and attempts to create archives for all the live messages it has missed since it went offline. In this case, the `timestamp` has to lie within the day the latest message was received and the current day.
+
+Exported messages MUST be restored as [14/WAKU2-MESSAGE](https://rfc.vac.dev/spec/14/) for bundling. Waku messages that are older than 30 days and have been exported for bundling can be removed from the control node's database (control nodes still maintain a database of application messages).
+
+# Message history archives
+
+Message history archives are represented as `WakuMessageArchive` and created from waku messages exported from the local database. Message history archives are implemented using the following protocol buffer.
```

Comment:
```suggestion
Message history archives are represented as `WakuMessageArchive` and created from waku messages exported from the local database. 
Message history archives are implemented using the following protocol buffer.
```

Reply:
Maybe link to the waku message spec here? :)

Reply:
```suggestion
Message history archives are represented as `WakuMessageArchive` and created from [waku messages](/spec/14/) exported from the local database. Message history archives are implemented using the following protocol buffer.
```